flanker-test-app/src/test/resources/task/flanker_all.json

```diff
@@ -0,0 +1,8112 @@
+{

```

Comment:
This is a copy of `/assets/task/Flanker.json`. Retrieving from `assets` would require this to be an AndroidTest, this was easier